docs/visualize/regionmap.asciidoc

```diff
@@ -0,0 +1,38 @@
+[[regionmap]]
+== Region Maps
+
+Region maps are thematic maps in which boundary vector shapes are colored using a gradient, with higher intensity colors indicating larger values and lower intensity colors indicating smaller values. 
+These are also known as choropleth maps. 
+
+image::images/regionmap.png[]
+
+
+=== Configuration
+
+To create such a map, you will need to configure a so-called "inner join", that will join the result of
+a terms-aggregation search-response from Elasticsearch to a reference vector file, based on a shared key.
+
```

Comment:
In general, stick to present tense. We usually opt for italics when introducing new terms instead of quotes.

To create a region map, you configure an _inner join_ that joins the result of an Elasticsearch terms aggregation  and a reference vector file based on a shared key.